src/internal_events/grpc.rs

```diff
@@ -9,11 +9,12 @@ pub struct GrpcInvalidCompressionScheme<'a> {
 
 impl InternalEvent for GrpcInvalidCompressionScheme<'_> {
     fn emit(self) {
-        error!(message = "Invalid compression scheme.",
-               error = ?self.status.message(),
-               error_type = error_type::REQUEST_FAILED,
-               stage = error_stage::RECEIVING,
-               internal_log_rate_secs = 10
+        error!(
+            message = "Invalid compression scheme.",
+            error = ?self.status.message(),
+            error_type = error_type::REQUEST_FAILED,
+            stage = error_stage::RECEIVING,
+            internal_log_rate_secs = true
```

Comment:
Almost :) The name was changed recently.

```suggestion
            internal_log_rate_limit = true
```

Reply:
darn it